build/Dockerfile

```diff
@@ -3,7 +3,8 @@ FROM quay.io/water-hole/ansible-operator
 COPY roles/ ${HOME}/roles/
 COPY playbooks/ ${HOME}/
 COPY watches.yaml ${HOME}/watches.yaml
+COPY _defaults.yml ${HOME}/_defaults.yml
```

Comment:
don't we need to use build/_defaults.yml here?

Reply:
You might be right. I considered the build directory to be the docker context for builds, but I am not exactly sure how the operator-sdk build works. I will test that tomorrow.

Reply:
I believe that everything invoked from the playbook should exist in the container. 
Maybe the build folder is not the best place for it logically (by operator-sdk semantics)

Reply:
Sure, that is what the COPY does. Maybe I should just move the defaults to the root directory?

Reply:
But we have the patch.yml and preprocessing script in build/ as well. And those are used for the same purpose (downstream customization)

Reply:
As long as they are copied i dont see a real problem